libpkgconf/client.c

```diff
@@ -78,7 +78,11 @@ pkgconf_client_init(pkgconf_client_t *client, pkgconf_error_handler_func_t error
 	pkgconf_path_build_from_environ("PKG_CONFIG_SYSTEM_INCLUDE_PATH", SYSTEM_INCLUDEDIR, &client->filter_includedirs, false);
 
 	/* GCC uses these environment variables to define system include paths, so we should check them. */
+#ifdef __HAIKU__
+	pkgconf_path_build_from_environ("BELIBRARIES", NULL, &client->filter_libdirs, false);
+#else
```

Comment:
you are certain GCC never checks `LIBRARY_PATH` on BeOS/Haiku?  that doesn't make much sense to me.  i would assume that `BELIBRARIES` is concatenated into `LIBRARY_PATH`, which means pkgconf_path_build_from_environ should be called on both env vars, in that order.

Reply:
OK.